src/pages/docs/messages/annotations.mdx

```diff
@@ -0,0 +1,418 @@
+</Aside>
+
+For more details on summary structure and usage, see [annotation summaries](#annotation-summaries).
+
+### Subscribing to individual annotations <a id="subscribe-individual-annotations" />
+
+For more fine-grained visibility, you can also subscribe directly to individual annotation events using the `annotations.subscribe()` method on a channel.
+
+Annotations delivered to the `annotations.subscribe()` listener will have an `action` of `annotation.create` or `annotation.delete`.
+
+<Code>
+```javascript
+await channel.annotations.subscribe((annotation) => {  
+  if (annotation.action === 'annotation.create') {
+    console.log(`New ${annotation.type} annotation with name ${annotation.name} from ${annotation.clientId}`);
+  } else if (annotation.action === 'annotation.delete') {
+    console.log(`${annotation.clientId} deleted a ${annotation.type} annotation with name ${annotation.name}`);
+  }
+});
+```
+
+```nodejs
+await channel.annotations.subscribe((annotation) => {  
+  if (annotation.action === 'annotation.create') {
+    console.log(`New ${annotation.type} annotation with name ${annotation.name} from ${annotation.clientId}`);
+  } else if (annotation.action === 'annotation.delete') {
+    console.log(`${annotation.clientId} deleted a ${annotation.type} annotation with name ${annotation.name}`);
+  }
+});
+```
+</Code>
+
+<Aside data-type='note'>
+Individual annotation events are useful for activity feeds or detailed logging, but for maintaining UI state, summary events are generally more reliable and efficient.
+</Aside>
+
+
+## Annotation types <a id="annotation-types"/>
+
+Annotation types determine how annotations are processed and aggregated into [summaries](#annotation-summaries).
+
+The annotation type is a string of the format `namespace:summarization.version` where:
+
+- `namespace` is a string (e.g. `reactions`) that groups related annotations. Only annotations in the same namespace will be aggregated together to produce [summaries](#annotation-summaries).
+- `summarization` specifies the summarization method (e.g. `flag`) which determines how annotations are aggregated to produce [summaries](#annotation-summaries).
+- `version` specifies the version component of the summarization method (e.g. `v1`), which allows for future changes to summarization behavior.
+
+## Annotation summaries <a id="annotation-summaries"/>
+
+When annotations for a message are published, Ably automatically generates a summary that provides an aggregated view of all annotations for that message.
+
+A separate summary is produced for each distinct [annotation type](#annotation-types).
+
+The summarization method specified in the [annotation type](#annotation-types) determines how annotations in the same namespace for a given message are aggregated into a summary. A summary is constructed from the set of `annotation.create` and `annotation.delete` messages that have been published for a message.
```

Comment:
Might be useful to give an example `summary` in a code block? Just to show that it's an object whose keys are the annotation types, which is obvious once you know it but isn't necessarily clear from reading the docs

Reply:
Fixed in https://github.com/ably/docs/pull/2642/commits/8d7eb0c7375d20f078fc2a0133bddc4eec64c697